feature-libs/organization/administration/core/store/effects/b2b-user.effect.ts

```diff
@@ -109,44 +120,64 @@ export class B2BUserEffects {
   updateB2BUser$: Observable<
     | B2BUserActions.UpdateB2BUserSuccess
     | B2BUserActions.UpdateB2BUserFail
+    | OrgUnitActions.AssignApprover
     | OrganizationActions.OrganizationClearData
   > = this.actions$.pipe(
     ofType(B2BUserActions.UPDATE_B2B_USER),
     map((action: B2BUserActions.UpdateB2BUser) => action.payload),
     filter((payload) => isValidUser(payload.userId)),
-    switchMap((payload) =>
-      this.b2bUserConnector
-        .update(payload.userId, payload.orgCustomerId, payload.orgCustomer)
+    switchMap(({ userId, orgCustomerId, orgCustomer }) => {
+      const isAssignedToApprovers = orgCustomer.isAssignedToApprovers;
+      return this.b2bUserConnector
+        .update(userId, orgCustomerId, orgCustomer)
         .pipe(
-          // TODO: change for 'payload: data' when backend API start to return user data on PATCH
-          switchMap(() => [new B2BUserActions.UpdateB2BUserSuccess(payload)]),
+          switchMap((_data) => {
+            const successActions = [
+              // TODO: change for 'payload: data' when backend API start to return user data on PATCH
+              new B2BUserActions.UpdateB2BUserSuccess(orgCustomer),
+              new OrganizationActions.OrganizationClearData(),
```

Comment:
`OrganizationActions.OrganizationClearData` is unnecessary here, as it will always be performed in `checkSelfEmailUpdate$` effect